docs/user-guides/workspace-access/jetbrains/index.md

```diff
@@ -210,163 +223,6 @@ cd /opt/idea/bin
-client files. We installed `nginx` and setup an FQDN and routed all requests to
-`/`. See below:
-
-```console
-server {
-        listen 80 default_server;
-        listen [::]:80 default_server;
-
-        root /var/www/html;
-
-        index index.html index.htm index.nginx-debian.html;
-
-        server_name _;
-
-        location / {
-                root /home/ubuntu;
-        }
-}
-```
-
-Then, configure your DNS entry to point to the IP address of the server. For the
-purposes of the POC, we did not configure TLS, although that is a supported
-option.
-
-### 4. Add Client Files
-
-You will need to add the following files on your local machine in order for
-Gateway to pull the backend and client from the server.
-
-```shell
-$ cat productsInfoUrl # a path to products.json that was generated by the backend's downloader (it could be http://, https://, or file://)
-
-https://internal.site/backends/<PRODUCT_CODE>/products.json
-
-$ cat clientDownloadUrl # a path for clients that you got from the clients' downloader (it could be http://, https://, or file://)
-
-https://internal.site/clients/
-
-$ cat jreDownloadUrl # a path for JBR that you got from the clients' downloader (it could be http://, https://, or file://)
-
-https://internal.site/jre/
-
-$ cat pgpPublicKeyUrl # a URL to the KEYS file that was downloaded with the clients builds.
-
-https://internal.site/KEYS
-```
-
-The location of these files will depend upon your local operating system:
-
-#### macOS
-
-```console
-# User-specific settings
-/Users/UserName/Library/Application Support/JetBrains/RemoteDev
-# System-wide settings
-/Library/Application Support/JetBrains/RemoteDev/
-```
-
-#### Linux
-
-```console
-# User-specific settings
-$HOME/.config/JetBrains/RemoteDev
-# System-wide settings
-/etc/xdg/JetBrains/RemoteDev/
-```
-
-#### Windows
-
-```console
-# User-specific settings
-HKEY_CURRENT_USER registry
-# System-wide settings
-HKEY_LOCAL_MACHINE registry
-```
-
-Additionally, create a string for each setting with its appropriate value in
-`SOFTWARE\JetBrains\RemoteDev`:
-
-![Alt text](../../images/gateway/jetbrains-offline-windows.png)
-
-### 5. Setup SSH connection with JetBrains Gateway
-
-With the server now configured, you can now configure your local machine to use
-Gateway. Here is the documentation to
-[setup SSH config via the Coder CLI](../../user-guides/workspace-access/index.md#configure-ssh).
-On the Gateway side, follow our guide here until step 16.
-
-Instead of downloading from jetbrains.com, we will point Gateway to our server
-endpoint. Select `Installation options...` and select `Use download link`. Note
-that the URL must explicitly reference the archive file:
-
-![Offline Gateway](../../images/gateway/offline-gateway.png)
-
-Click `Download IDE and Connect`. Gateway should now download the backend and
-clients from the server into your remote workspace and local machine,
-respectively.
-
 ## JetBrains Fleet
```

Comment:
thoughts about extracting Jetbrains Fleet to a separate page too?

Reply:
maybe - I'm still working through how this should flow and where Fleet fits in

just wanted to push this up and test the linting 😁

Reply:
I want us to have a similar page for zed.dev editor it would be similar to Fleet instructions.